pkg/koordlet/metricsadvisor/framework/config.go

```diff
@@ -48,7 +50,9 @@ func NewDefaultConfig() *Config {
 		PSICollectorInterval:             10 * time.Second,
 		CPICollectorTimeWindow:           10 * time.Second,
 		ColdPageCollectorInterval:        5 * time.Second,
+		ResctrlCollectorInterval:         1 * time.Second,
```

Comment:
now 1 second seems to short and not necessary now, how about 10 seconds by default?